exhibitions/src/main/res/layout/fragment_all_exhibitions.xml

```diff
@@ -7,32 +7,38 @@
     android:layout_height="match_parent"
     android:animateLayoutChanges="true">
 
-    <android.support.design.widget.AppBarLayout
-        android:id="@+id/appBarLayout"
+    <android.support.v7.widget.Toolbar
```

Comment:
similar to above lets create a layout we include for this and use in the three fragments. At least for that PR. After that we can use it in other places as well

Reply:
This is done too! It's in commit e68b240b, merged in #299.